test/integration/targets/podman_container/tasks/main.yml

```diff
@@ -0,0 +1,297 @@
+          - "9001:8000"
+        workdir: "/bin"
+        env:
+          FOO: bar
+          BAR: foo
+        group_add: "somegroup"
+        label:
+          somelabel: labelvalue
+          otheralbe: othervalue
+        volumes:
+          - /tmp:/data
+      register: test
+
+    - name: Check output is correct
+      assert:
+        that:
+          - test is changed
+          - test.ansible_facts is defined
+          - test.ansible_facts.podman_container is defined
+          - test.ansible_facts.podman_container != {}
+          - test.ansible_facts.podman_container['State']['Running'] == true
+          # test capabilities
+          - "'CAP_SYS_TIME' in test.ansible_facts.podman_container['BoundingCaps']"
+          - "'CAP_NET_ADMIN' in test.ansible_facts.podman_container['BoundingCaps']"
+          # test annotations
+          - test.ansible_facts.podman_container['Config']['Annotations']['this'] is defined
+          - test.ansible_facts.podman_container['Config']['Annotations']['this'] == "annotation_value"
+          # test DNS
+          - test.ansible_facts.podman_container['HostConfig']['DNS'] is defined
+          - test.ansible_facts.podman_container['HostConfig']['DNS'] == ['1.1.1.1', '8.8.4.4']
+          # test ports
+          - test.ansible_facts.podman_container['NetworkSettings']['Ports']|length == 2
+          # test working dir
+          - test.ansible_facts.podman_container['Config']['WorkingDir'] == "/bin"
+          # test dns search
+          - test.ansible_facts.podman_container['HostConfig']['DNSSearch'] == ['example.com']
+          # test environment variables
+          - "'FOO=bar' in test.ansible_facts.podman_container['Config']['Env']"
+          - "'BAR=foo' in test.ansible_facts.podman_container['Config']['Env']"
+          # test labels
+          - test.ansible_facts.podman_container['Config']['Labels'] | length == 2
+          - test.ansible_facts.podman_container['Config']['Labels']['somelabel'] == "labelvalue"
+          - test.ansible_facts.podman_container['Config']['Labels']['otheralbe'] == "othervalue"
+          # test mounts, in some of podman versions "source" and "destination" could be with different case
+          # that's totally bizarre, but that's what it is
+          - >-
+            test.ansible_facts.podman_container['Mounts'][0]['destination'] is defined and
+            '/data' in test.ansible_facts.podman_container['Mounts'] | map(attribute='destination') | list or
+            test.ansible_facts.podman_container['Mounts'][0]['Destination'] is defined and
+            '/data' in test.ansible_facts.podman_container['Mounts'] | map(attribute='Destination') | list
+          - >-
+            test.ansible_facts.podman_container['Mounts'][0]['source'] is defined and
+            '/tmp' in test.ansible_facts.podman_container['Mounts'] | map(attribute='source') | list or
+            test.ansible_facts.podman_container['Mounts'][0]['Source'] is defined and
```

Comment:
I don't think that supporting two versions of the same param spelled differently is a good idea.

Reply:
The problem that it depends on OS also, in centos7 and fedora29 it has older version with upper case, on rhel8 and fedora30 it has newer version with lower case. So if I don't cover it, tests will fail on two of OS. 
Maybe I'll leave there TODO to remove it when it has same version everywhere

Reply:
So maybe just normalize the structs to be the same in all cases?

Reply:
@webknjaz btw, do you have idea how to normalize it here? adding map('lower') or kind of?

Reply:
Are you sure the newer version uses lower-case? According to the source: https://github.com/containers/libpod/blob/master/libpod/container_inspect.go#L114 I think it was added in containers/libpod#3312

Reply:
older version uses lower case: 
```
Version:            1.3.2
RemoteAPI Version:  1
Go Version:         go1.10.3
OS/Arch:            linux/amd64
```

```
{
                "destination": "/tmp",
                "type": "bind",
                "source": "/tmp",
                "options": [
                    "rbind",
                    "rw",
                    "rprivate"
                ]
            },
```